Cargo.toml

```diff
@@ -185,7 +185,7 @@ syn = { version = "2.0.90" }
 synstructure = { version = "0.13.1" }
 # This dependency was added through git since we are experimenting with a fork of sysinfo which adds additional
 # capabilities of reading cgroups memory stats. Such stats are used in Relay to correctly determine memory usage.
-sysinfo = { git = "https://github.com/getsentry/sysinfo.git", rev = "e2e5d530600f96bdd79652c856918da23e5dd938" }
+sysinfo = { git = "https://github.com/getsentry/sysinfo.git", rev = "15b3be3273ba286740122fed7bb7dccd2a79dc8f" }
```

Comment:
Since the PR is already merged, we can link directly to the upstream repo, update the comment to mention that we're just waiting for a release?

Reply:
It seems to not work if omit `rev` or just add `branch = "master"`. I have no idea why :/